app/_locales/en/messages.json

```diff
@@ -3800,7 +3800,7 @@
     "message": "Restore user data"
   },
   "restoreUserDataDescription": {
-    "message": "You can restore user settings containing preferences and account addresses from a previously backed up JSON file."
+    "message": "You can restore data like contacts and preferences from a backed up JSON file."
```

Comment:
> ... from a backup file.

Reply:
@coreyjanssen wdyt?

Reply:
Also agree! `You can restore data like contacts and preferences from a backup file. `

Reply:
Solved in [f2cb351](https://github.com/MetaMask/metamask-extension/pull/21715/commits/f2cb3516c8af059835c4b8adfe8d24b20c433950)